src/plugins/workspace/server/routes/index.ts

```diff
@@ -255,4 +258,19 @@ export function registerRoutes({
       return res.ok({ body: result });
     })
   );
+
+  router.get(
+    {
+      path: `${WORKSPACES_API_BASE_URL}/is_workspace_enabled`,
+      validate: {},
+    },
+    router.handleLegacyErrors(async (context, req, res) => {
```

Comment:
Should it call `context.core.uiSettings.client.get(FEATURE_FLAG_KEY_IN_UI_SETTING)` to get the flag directly from saved objects?

Reply:
I think that the flag in browser side should be consistent to the flag in server side. And the flag in server side may not be consistent to the flag in kibana index because user may call index update API to update the flag directly.

Reply:
I see, so if changing the flag in the index but not via `setWorkspaceFeatureFlag` will require a server restart?

Reply:
Yes, and that is not a expected behavior to update flag.